spec/features/medicaid_application_with_maximum_info_spec.rb

```diff
@@ -104,11 +104,11 @@
     on_pages "Income & Expense Amounts" do
       click_on "Next"
 
-      fill_in "Your Current Job (#1)", with: 100
-      fill_in "Your Current Job (#2)", with: 50
-      fill_in "Your Current Job (#3)", with: 25
-      fill_in "Your Self-Employment", with: 100
-      fill_in "Your Unemployment", with: 100
+      fill_in "step_employed_monthly_income_0", with: 100
```

Comment:
Are we keying off of ID to reduce reliance of our tests on copy? Or is there another reason?

Reply:
the reason is that the input labels are now non-unique so the input ids were better for targeting specific inputs. I liked the readability of what we had before but this does have the added benefit of being *not* tied to copy!